src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,43 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    private static final int MAX_LENGTH = 100;
+    private static final int FIGURE_COUNT = 5;
+    private static final int DEFAULT_RADIUS = 10;
+    private static final String DEFAULT_COLOR = String.valueOf(Color.WHITE);
```

Comment:
Use name() for getting String representation of enum constants
Don’t use toString() or String.valueOf()(it will call toString() under the hood) for getting String representation of enum constants. toString() is common for all enum constants. If you override this method like below:
 ```
@Override
 public String() toString() {
    return "default";
 }
```
then for every constant toString() will be returning default, that’s not ok. So it’s better to use standard method of enum name() that will be returning always String representation of concrete enum constant.

Reply:
![image](https://github.com/mate-academy/jv-oop-advanced/assets/138139466/d4f8406b-9b8e-4d97-9022-e858863b9e50)